elisa/src/org/labkey/elisa/ElisaProviderSchema.java

```diff
@@ -17,25 +17,6 @@
 {
     public static final String CURVE_FIT_METHOD_TABLE_NAME = "CurveFitMethod";
 
-    enum ElisaCurveFits
-    {
-        FOUR_PARAMETER(StatsService.CurveFitType.FOUR_PARAMETER.getLabel()),
-        FIVE_PARAMETER(StatsService.CurveFitType.FIVE_PARAMETER.getLabel()),
```

Comment:
@labkey-klum  Any reason we need/want to leave the 5PL type around for the single plate assay case? I believe it was their prior to this work. I have a local 20.7 MSSQL DB which has a run that had Curve Fit Method = Five Parameter. The upgraded/populated CurveFit table for that container looks like this:
<img width="1416" alt="Screen Shot 2020-10-15 at 11 47 41 AM" src="https://user-images.githubusercontent.com/6411206/96160987-5107ed00-0edc-11eb-8dfa-3971062e0793.png">

Reply:
I decided to remove it because it didn't yield good fits. I think if we wanted to add a 5PL fit in the future we would use the new 4PL simplex algorithm.

Reply:
That sounds good. Just checking. Thanks.